File: python/raindrops/raindrops.py
def convert(number):
    conc_dict = {3: "Pling", 5: "Plang", 7: "Plong"}

    hold = ""
    for fact, name in conc_dict.items():
        if number % fact == 0:
            hold += name

    return hold or str(number)

File: go/raindrops/raindrops.go
package raindrops

import "strconv"

// Convert turns an integer into a string indicating raindrop sound.
func Convert(num int) string {
	var result = ""
	if num%3 == 0 {
		result += "Pling"
	}
	if num%5 == 0 {
		result += "Plang"
	}
	if num%7 == 0 {
		result += "Plong"
	}
	if result == "" {
		result = strconv.Itoa(num)
	}
	return result
}
